compiler/rustc_data_structures/src/sync.rs

```diff
@@ -43,46 +43,6 @@ cfg_if! {
         use std::ops::Add;
         use std::panic::{resume_unwind, catch_unwind, AssertUnwindSafe};
 
-        /// This is a single threaded variant of AtomicCell provided by crossbeam.
-        /// Unlike `Atomic` this is intended for all `Copy` types,
-        /// but it lacks the explicit ordering arguments.
-        #[derive(Debug)]
-        pub struct AtomicCell<T: Copy>(Cell<T>);
```

Comment:
There is a re-export of `AtomicCell` from crossbeam for `#[cfg(feature = "parallel_rustc")]`, right? That one should probably be removed.